ts/att/time.ts

```diff
@@ -0,0 +1,23 @@
+export class Time {
+
+    public static CreateTime24Hour(hour: number, minute: number): Time {
+        return new Time(hour, minute);
+    }
+
+    public Hour: number;
+    public Minute: number;
+
+    private constructor(hour: number, minute: number) {
+        this.Hour = hour;
+        this.Minute = minute;
+    }
+    public Equal(other: Time): boolean {
+        if (this.Hour !== other.Hour) {
```

Comment:
@DheenodaraRao Don't use brackets if the if body contains one line, so this line should be formatted as 
```ts
if(this.Hour !== other.Hour) return false;
if(this.Minute !== other.Minute) return false;
return true;
```

Reply:
if (this.Hour !== other.Hour) return false;
    if (this.Minute !== other.Minute) return false;

Reply:
tslint gives me error if I dont put the bracket.